app/views/pages/index.html.erb

```diff
@@ -37,9 +37,9 @@
         <span>Find and fix flaky tests</span>
         <ul>
           <li><%= link_to "Overview", "/docs/test-analytics" %></li>
-          <li><%= link_to "Deep performance analysis", "/docs/test-analytics#spot-sneaky-slow-downs" %></li>
-          <li><%= link_to "Find and fix flaky tests", "/docs/test-analytics#find-flaky-tests" %></li>
-          <li><%= link_to "Speed and reliability monitoring", "/docs/test-analytics#monitor-what-matters" %></li>
+          <li><%= link_to "Deep performance analysis", "/docs/test-analytics/features#spot-sneaky-slow-downs" %></li>
+          <li><%= link_to "Find and fix flaky tests", "/docs/test-analytics/features#find-flaky-tests" %></li>
```

Comment:
```suggestion
          <li><%= link_to "Find and fix flaky tests", "/docs/test-analytics/features#find-flaky-unreliable-tests" %></li>
```

🚨 Fix test for broken section/anchor link, as the heading has been changed on `/test-analytics/features`